include/eve/traits/overload/default_behaviors.hpp

```diff
@@ -28,18 +29,28 @@ namespace eve
   {
     using base = decorated_with<OptionsValues, Options...>;
 
-    template<typename T>
-    constexpr auto operator[](T t) const
-    requires( requires(base const& b) { b[t];} )
+    template<callable_options O> constexpr auto operator[](O const& opts) const
+    {
+      return  Func<O>{opts};
+    }
```

Comment:
This seems weird. In algo we merge old a new options here, preferring new

Reply:
This was probably written  so it worked for abs and I didn't think about that. I can amend the way you want

Reply:
Plese merge, preferring the new to old. At least for now.

Reply:
Will do

Reply:
Turns out it was uneeded.